City-of-Helsinki/drupal-helfi-platform-config
# UHF-2511 Translations

How to test: 
- Fetch db from Sote production proceeding (or ask @khalima)
- Checkout `UHF-2511_translations` for the Sote drupal instance
- Run `composer require drupal/helfi_platform_config:dev-UHF-2511_translations && composer require drupal/hdbt:dev-UHF-2511_translations && composer require drupal/helfi_tpr:dev-UHF-2511_translations`
- Run `make fresh`
- **Check that the following information has been translated**
- Go to https://helfi-sote.docker.so/en/health-and-social-services/health-care/health-stations/oulunkyla-health-station, https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/terveydenhoito/terveysasemat/oulunkylan-terveysasema and https://helfi-sote.docker.so/sv/social-och-halsovardstjanster/halsovard/halsostationer/aggelby-halsostation
   - [ ] Contact information | Yhteystiedot | Kontaktuppgifter
   - [ ] Visiting address | Käyntiosoite | Besöksadress
   - [ ] Telephone number | Puhelinnumero | Telefonnummer
   - [ ] Opening hours | Aukioloajat | Öppettider
   - [ ] Postal address | Postiosoite | Postadress
   - [ ] Services | Palvelut | Tjänster
   - [ ] 1 service / 2+ services | 1 palvelu / 2+ palvelua | 1 tjänst / 2+ tjänster
   - [ ] Accessibility information | Esteettömyystiedot | Tillgänglighetsuppgifter
- Go to https://helfi-sote.docker.so/en/health-and-social-services/health-care/health-stations/nurses-appointment-at-a-health-station, https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/terveydenhoito/terveysasemat/hoitajan-vastaanotto-terveysasemalla and https://helfi-sote.docker.so/sv/social-och-halsovardstjanster/terveydenhoito/terveysasemat/vardarmottagning-pa-halsostationen
   - [ ] Requires authentication | Vaatii tunnistautumisen | Kräver identifiering
   - [ ] Load more service points | Tärkeitä linkejä | Ladda fler serviceställen
   - [ ] Service points | Palvelupaikat | Serviceställen
   - [ ] service point / service points | palvelupaikka / palvelupaikkaa | serviceställe / serviceställen
   - [ ] Load more service points | Lataa lisää palvelupaikkoja | Ladda fler serviceställen
- Finally run `make drush-cex` and check that the there are no massive changes to configuration files. Before this update there were 60 changes to configuration files with just initial config export.

Check also these: 
- https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/67
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/166
- https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/178
- https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/150
- https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/129
